test/shared/setup.ts

```diff
@@ -16,7 +22,7 @@ export interface Fossil {
 }
 
 export async function vanillaSetup() {
-  const controller = (await starknet.deployAccount('OpenZeppelin')) as Account;
```

Comment:
why the change from OZ to Argent? :)

Reply:
Made more sense I thought as we would be using argent wallets to interact with SX